luigi/server.py

```diff
@@ -73,6 +92,62 @@ def get_template_path(self):
         return pkg_resources.resource_filename(__name__, 'templates')
 
 
+class AllRunHandler(BaseTaskHistoryHandler):
+
+    def get(self):
+        # get task history db in the same path
+        engine = create_engine('sqlite:///.task_history.db', echo=True)
+        conn_sqlite = engine.connect()
+        query = "select id, name from tasks order by id desc"
+        task_list = conn_sqlite.execute(query)
+        tasknames = []
+        for task in task_list:
+            tasknames.append(task['name'])
+        self.render("menu.html", tasknames=tasknames)
+
+
+class SelectedRunHandler(BaseTaskHistoryHandler):
+
+    def get(self, name):
+        tasks = {}
+        chartResults = {}
+        tableResults = []
+        # get task history db in the same path
+        engine = create_engine('sqlite:///.task_history.db', echo=True)
```

Comment:
I came back to this work and commented to the html files. But after the version upgrading of luigi, I faced with the db connection issue. I'm trying to follow the current way that luigi connects to db with sqlalchemy.